kmir/src/kmir/kdist/mir-semantics/rt/data.md

```diff
@@ -190,17 +190,13 @@ Projections operate on the data stored in the `TypedLocal` and are therefore spe
     requires PROJECTIONS =/=K .ProjectionElems
      andBool 0 <=Int I
      andBool I <Int size(LOCALS)
-
-  rule <k> #readOperand(operandMove(place(local(I), PROJECTIONS) #as PLACE)) ~> CONT
-        =>
-           #LocalError(Unsupported("Moving operand with projection")) ~> PLACE ~> CONT
-       </k>
-       <locals> LOCALS </locals>
-    requires PROJECTIONS =/=K .ProjectionElems
-     andBool 0 <=Int I
-     andBool I <Int size(LOCALS)
 ```
 
+When an operand is `Moved` by the read, the original has to be invalidated. In case of a projected value, this is a write operation nested in the data that is being read.
+In contrast to regular write operations, the value does not have to be _mutable_ in order to get moved,
+so we need to copy the respective code for writing, or make it generic.
+
+Related code currently resides in the value-implementing module.
```

Comment:
This is a bit unfortunate... 
I am using the `#updateProjections` function in the high-level module. That function could maybe also move into this module here (because now we have the `Aggregate` constructor in the low-level data, too).